src/components/Icon/Icon.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+export interface IconProps {
```

Comment:
Interfaces vs Type. I prefer type for everything. If you don't, when to use interface and when type?

Reply:
I use interface for everything 😄

Reply:
I often use interface for objects and type for union types and such

Reply:
so that makes 'use interface whenever possible, else type'